services/frontend/pkg/command/server.go

```diff
@@ -90,6 +90,9 @@ func Server(cfg *config.Config) *cli.Command {
 				logger.Fatal().Err(err).Msg("failed to register the http service")
 			}
 
+			// start event handler
+			go ListenForEvents(cfg, logger)
```

Comment:
I am not entirely sure, but shouldn't this be part of the `run.Group{}`? 

Also I think we should pass `ctx` to `ListenForEvent()` and implement proper cancellation via `context.Done()` in `ListenForEvents()`. Otherwise the go-routine can't be properly ended.